metrics-delivery-endpoint-api/src/main/java/eu/openaire/mas/delivery/MetricsDelivery.java

```diff
@@ -8,8 +10,13 @@
  */
 public interface MetricsDelivery {
 
-    MetricEntry deliver(String groupId, String metricId, String from, String to);
+    MetricEntry deliver(String groupId, String metricId);
```

Comment:
Sure, we can do this in a separate ticket. 

I think since they are in this package/module they will be mostly bound to KPIs context. Don't think we will use it anywhere else.